cmd/create/create.go

```diff
@@ -31,5 +31,6 @@ var Cmd = &cobra.Command{
 func init() {
 	Cmd.AddCommand(port.Cmd)
 	Cmd.PersistentFlags().StringVarP(&pkg.Options.InstanceID, "instance-id", "i", "", "Instance ID of the PowerVS instance")
-	_ = Cmd.MarkPersistentFlagRequired("instance-id")
+	Cmd.PersistentFlags().MarkDeprecated("instance-id", "instance-id is deprecated, workspace-id should be used")
+	Cmd.PersistentFlags().StringVarP(&pkg.Options.InstanceID, "workspace-id", "", "", "Workspace ID of the PowerVS workspace")
```

Comment:
lets rename the field name from `Options.InstanceID` to `Options.WorkspaceID`

Reply:
ok, will update.